fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/ChunkSectionLayerMap.java

```diff
@@ -17,39 +17,38 @@
 package net.fabricmc.fabric.api.client.rendering.v1;
 
 import net.minecraft.client.renderer.chunk.ChunkSectionLayer;
-import net.minecraft.client.renderer.rendertype.RenderTypes;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.material.Fluid;
 
-import net.fabricmc.fabric.impl.client.rendering.BlockRenderLayerMapImpl;
+import net.fabricmc.fabric.impl.client.rendering.ChunkSectionLayerMapImpl;
 
 /**
- * Use to associate blocks or fluids with block render layer other than default (solid).
+ * Use to associate blocks or fluids with a chunk section layer other than default (solid).
  *
- * <p>{@link RenderTypes} control how sprite pixels for fluids and blocks are blended with the scene. Consult the
- * vanilla {@link RenderTypes} implementation for examples.
+ * <p>{@link ChunkSectionLayer} control how sprite pixels for fluids and blocks are blended with the scene. Consult the
+ * vanilla {@link ChunkSectionLayer} implementation for examples.
  *
  * <p>The Fabric Renderer API can be used to control this at a per-quad level via {@code BlendMode}.
```

Comment:
```suggestion
 * <p>The Fabric Renderer API can be used to control this at a per-quad level.
```

Not related to this PR but better to do now so we don't forget.